### PR TITLE
Yellowstone ms

### PR DESCRIPTION
**Pull request by michaelsanchez-pingidentity**

### CODE REVIEW

File: bxhealth/src/components/Controller/DauthZ.js [hunk 1]
```diff
@@ -31,6 +31,20 @@ class DauthZ {
         return await menuResponse;
     }
 
+    /**
+    Retreive caregiver Menu:
+    Gets menu/feature options to manage a patient's healthcare.
+
+    @param {string} token Access token from the authenticated caregiver.
```
Comment (jon-oblander) [inline bxhealth/src/components/Controller/DauthZ.js:38]: patient param needed

File: bxhealth/src/components/Integration/PingAuthorize.js [hunk 2]
```diff
@@ -36,6 +36,32 @@ class PingAuthorize {
         
         return await responseJSON;
     }
+
+    /**
+    Retreive caregiver Menu:
+    Fetches patient healthcare menu options for the caregiver.
+    
+    @param {string} token Access token from the authenticated patient.
+    @param {string} child Given name of the child for which to retreive menu options.
```
Comment (jon-oblander) [inline bxhealth/src/components/Integration/PingAuthorize.js:45]: child param should be patient

File: bxhealth/src/components/Controller/DauthZ.js (hunk 1 above)
Comment (michaelsanchez-pingidentity) [reply bxhealth/src/components/Controller/DauthZ.js:38]: Good catch. Added.

File: bxhealth/src/components/Integration/PingAuthorize.js (hunk 2 above)
Comment (michaelsanchez-pingidentity) [reply bxhealth/src/components/Integration/PingAuthorize.js:45]: Classic copy paste. Good eye.

**Review by jon-oblander — APPROVED** [review]
reviewed

**Pull request merged**